Pursuit-Core-iOS-Episodes-from-Online/Pursuit-Core-iOS-Episodes-from-Online/Controllers/DetailViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  DetailViewController.swift
+//  Pursuit-Core-iOS-Episodes-from-Online
+//
+//  Created by Malcolm S. Turnquest on 9/12/19.
+//  Copyright © 2019 Benjamin Stone. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+
+    var episode: Episodes?
+        @IBOutlet weak var detailImage: UIImageView!
+        
+        @IBOutlet weak var episodeDetail: UILabel!
+        
+        @IBOutlet weak var episodeNum: UILabel!
+        
+        @IBOutlet weak var episodeDescription: UITextView!
+        override func viewDidLoad() {
+            super.viewDidLoad()
+            loadLabel()
+            loadImage()
+        }
+        func loadLabel(){
```

Comment:
Mark these functions private, because they are exclusive to this view controller.